public/index.html

```diff
@@ -24,10 +24,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
-    <style>
-      @import url('https://fonts.googleapis.com/css2?family=Work+Sans:ital,wght@0,100;0,200;0,300;0,400;0,600;1,100;1,200;1,300&display=swap');
-      </style> 
+    <title>MINDRESET</title>
```

Comment:
Where's the button and how long do I need to push it :D

Reply:
the button ist only in your mind :D